tap_azure_git/__init__.py

```diff
@@ -1106,7 +1106,8 @@ def get_all_pipeline_runs(schema, org, repo_path, pipeline, state, mdata, start_
             run = {
                 **raw_run,
                 '_sdc_repository': sdc_repository,
-                '_sdc_id': '{}/pipeline/{}/run/{}'.format(sdc_repository, pipeline['id'], raw_run['id'])
+                '_sdc_id': '{}/run/{}'.format(sdc_repository, raw_run['id']),
```

Comment:
These changes to `_sdc_id` can't happen because the `run['id']` is not unique.  Each pipelines runs would overwrite the others if we don't include the pipeline id.  Same thing goes for the build and timelines.

I will fix these before I get this merged and updated for all orgs. @myleshenderson